lib/transport/selenium-webdriver/selenium.js

```diff
@@ -10,21 +9,24 @@ module.exports = class SeleniumServer extends DefaultSeleniumDriver {
     const opts = new Options({settings});
     opts.updateWebdriverPath();
 
-    const seleniumService = new SeleniumServerBuilder(settings);
-    const moduleKey = settings.webdriver.log_file_name || '';
+    const seleniumService = new this.ServiceBuilder(settings);
 
-    seleniumService.setOutputFile(moduleKey);
+    const outputFile = settings.webdriver.log_file_name || '';
+    seleniumService.setOutputFile(outputFile);
 
     return seleniumService;
+  }
 
+  static get ServiceBuilder() {
+    return require('./service-builders/selenium.js');
   }
```

Comment:
We don't need to add a static method. Create a new class for Appium instead and extend this class over there.